### ISSUE TITLE
iMusic - The Perfect Music Player - Listen to Free Music Without Even Touching Your Screen

### ISSUE THREAD

**Issue by Scholle3000** [glasklart/hd#5166]
**App Name:** iMusic - The Perfect Music Player - Listen to Free Music Without Even Touching Your Screen
**Bundle ID:** com.SBAPPS.imusic
**iTunes ID:** <a target="_blank" href="http://getart.dreamnet.at?id=487677479">487677479</a>
**iTunes URL:** <a target="_blank" href="https://itunes.apple.com/us/app/imusic-perfect-music-player/id487677479?mt=8&uo=4">https://itunes.apple.com/us/app/imusic-perfect-music-player/id487677479?mt=8&uo=4</a>
**App Version:** 5.2
**Seller:** Xhavit Mustafa
**Developer:** <a target="_blank" href=https://itunes.apple.com/us/artist/xhavit-mustafa/id928951014?uo=4>© Xhavit Mustafa</a>
**Supported Devices:** iPhone5c, iPhone6, iPhone5s, iPhone6Plus, iPadThirdGen4G, iPad2Wifi, iPadMini4G, iPhone4S, iPadFourthGen4G, iPhone5, iPad23G, iPadThirdGen, iPadFourthGen, iPadMini, iPodTouchFifthGen
**Original Artwork:**
<img src="http://a1337.phobos.apple.com/us/r30/Purple1/v4/81/1c/6d/811c6d56-69bb-6ab8-5da4-f260c75abcc1/mzl.wtkxjrie.png" width="150" height="150" />
**Accepted Artwork:**
![](http://preview.dreamnet.at/?image=https://cloud.githubusercontent.com/assets/10515074/6113809/262097e6-b09a-11e4-8620-a5cd7a02e36d.png)

**Comment by andreolidavide** [comment]
![](http://preview.dreamnet.at/?image=https://cloud.githubusercontent.com/assets/10515074/6105966/5e19a45e-b05d-11e4-9fac-93fcbd150018.png)
https://cloud.githubusercontent.com/assets/10515074/6105966/5e19a45e-b05d-11e4-9fac-93fcbd150018.png

Source
https://cloud.githubusercontent.com/assets/10515074/6105990/96eaef54-b05d-11e4-8ab8-892dbd59dc0a.png

**Comment by Scholle3000** [comment]
Thx. for the rapid creation looks very fine

**Comment by MagnaMike** [comment]
@andreolidavide Hmm, can you just get rid of the border, it seems unnecessary?

**Comment by andreolidavide** [comment]
@MagnaMike Yes of course, no problem
![](http://preview.dreamnet.at/?image=https://cloud.githubusercontent.com/assets/10515074/6113809/262097e6-b09a-11e4-8620-a5cd7a02e36d.png)
https://cloud.githubusercontent.com/assets/10515074/6113809/262097e6-b09a-11e4-8620-a5cd7a02e36d.png

Source
https://cloud.githubusercontent.com/assets/10515074/6113817/39fd9a7a-b09a-11e4-8eec-76572c6d6dd5.png

**Comment by MagnaMike** [comment]
@andreolidavide Looks good. Let's go with that one. :+1:

**Comment by andreolidavide** [comment]
@MagnaMike do you mean like this?https://cloud.githubusercontent.com/assets/10515074/6113990/5afff852-b09b-11e4-9503-fc2cb27e73ae.png

**Comment by MagnaMike** [comment]
@andreolidavide Perfect. :ok_hand:

**Comment by PolarisBeaver** [comment]
@andreolidavide @MagnaMike @ivanov007 there seems to be a small error with the source file resolution, it was 1040x1024, but I have corrected that here with minimal alteration. https://cloud.githubusercontent.com/assets/10394552/6118302/1ec1405a-b071-11e4-913f-a04d68c51dd0.png

**Comment by dreamnet** [comment]
@MagnaMike @andreolidavide @PolarisBeaver The 1024x1024px of the source file represents the 78x78px area in the center of the finished Glasklart icon.
This means if you make the icon in the source file going to the edges, the result for this icon will look like this:
![2x](http://preview.dreamnet.at?image=https://cloud.githubusercontent.com/assets/2068130/6174112/078f466a-b2ed-11e4-9213-0e534d56c9bc.png)
If you make the icon in the source file smaller (as @andreolidavide originally did) the result would look like this:
![2x](http://preview.dreamnet.at?image=https://cloud.githubusercontent.com/assets/2068130/6174225/a8656a2e-b2ed-11e4-80a7-aa697184f30a.png)
...and that's what we want for this icon.
In other words, the 1024x1024px source file will be resized to 78x78px and then the Glasklart style will be applied.
So the first source file from @andreolidavide without the canvas is correct.

**Comment by MagnaMike** [comment]
My mistake @dreamnet. I had misunderstood what the sources were going to be used for.

**Comment by dreamnet** [comment]
@MagnaMike No problem Mike! :+1:
